classes/class-wc-connect-nux.php

```diff
@@ -156,6 +156,29 @@ public function register_order_page_labels_pointer( $pointers ) {
 			return $pointers;
 		}
 
+		/**
+		 * Check that the current user is the owner of the Jetpack connection
+		 * - Only that person can accept the TOS
+		 *
+		 * @uses self::get_jetpack_install_status()
+		 *
+		 * @return bool
+		 */
+		public function can_accept_tos() {
+			// Developer case
+			if ( self::JETPACK_DEV === $this->get_jetpack_install_status() ) {
+				return true;
+			}
+
+			$user_token = Jetpack_Data::get_access_token( JETPACK_MASTER_USER );
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-services/pull/1062/commits/d7b41835a0108fe57426d75f2cf092473f73876c.